src/Network/Network/FirewallPolicy/PolicySettings/AzureApplicationGatewayFirewallPolicySetting.cs

```diff
@@ -43,6 +51,10 @@ public class AzureApplicationGatewayFirewallPolicySetting : NetworkBaseCmdlet
         [ValidateNotNullOrEmpty]
         public int MaxRequestBodySizeInKb { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Whether allow WAF to enforce file upload limits.")]
```

Comment:
Change to "Allow WAF to enforce file upload limit". Do we not specific default value here?